index.html

```diff
@@ -674,13 +693,16 @@ <h3 class="contact_title">location</h3>
                 cols="1"
                 rows="7"
                 class="contact_input"
+                required
+                placeholder="enter your message"
+                maxlength="500"
               ></textarea>
             </div>
             <div>
-              <a href="#" class="button button--flex">
+              <button type="submit" class="button button--flex">
```

Comment:
please fix the `button--flex` class name. The general convention we are following is either a camelcase or a snake case